### ISSUE TITLE
code

### ISSUE THREAD

**Issue by zyfsa** [bonlime/keras-deeplab-v3-plus#19]
hi, bonlime. can you release your complete code? thanks ,I write our own Loss and metrics, but I don not know if it is right. The result is bad

**Comment by bonlime** [comment]
I can't share complete code right now, but here is my loss 

```
import keras.backend as K
from keras.losses import binary_crossentropy
def jaccard_coef(y_true, y_pred):
        y_true_f = K.flatten(y_true)
        y_pred_f = K.flatten(y_pred)
        intersection = K.sum(y_true_f * y_pred_f)
        return (intersection + 1.0) / (K.sum(y_true_f) + K.sum(y_pred_f) - intersection + 1.0)

def bce_dice_loss(y_true,y_pred):
        return binary_crossentropy(y_true, y_pred) + 1. - jaccard_coef(y_true, y_pred)
```
you can use bce_dice_loss as loss for training, and jaccard as metrics

**Comment by bhack** [comment]
@bonlime Do you have used F metric? See https://github.com/tensorflow/models/issues/3792

**Comment by bonlime** [comment]
@bhack No, I didn't

**Comment by zyfsa** [comment]
thank you,can you say the size of your y_true and y_pred, the form is? represpent is? Besides, can you share your metrics. please , I do not know why my result is pool

**Comment by srihari-humbarwadi** [comment]
@bonlime 
```
def jaccard_coef(y_true, y_pred):
    y_true_f = K.flatten(y_true)
    y_pred_f = K.flatten(y_pred)
    intersection = K.sum(y_true_f * y_pred_f)
    return (intersection + 1.0) / (K.sum(y_true_f) + K.sum(y_pred_f) - intersection + 1.0)

def bce_dice_loss(y_true,y_pred):
    return binary_crossentropy(y_true, y_pred) + 1. - jaccard_coef(y_true, y_pred)
``` 
can i replace bce with categorical_crossentroppy for multiclass problems ?
